### PR TITLE
B2B-1157 Transfer from Oauth broker (i.e. Coinbase) flow breaks in a RN SDK app

### PR DESCRIPTION
**Pull request by alex-zemlyanukhin-mesh**
https://front-finance.atlassian.net/browse/B2B-1157

### CODE REVIEW

**Review by vitalii-movchan-mesh — COMMENTED** [review]
Plz use correct title
Plz attach Jira link in PR description
Plz resolve conflicts

**Comment by alex-zemlyanukhin-mesh** [comment]
> Plz use correct title Plz attach Jira link in PR description Plz resolve conflicts

done

File: packages/front-b2b-link-rn/src/Front.tsx [hunk 1]
```diff
@@ -55,10 +56,12 @@ const FrontFinance = (props: {
     ) {
       setShowWebView(false)
     }
-    if (type === 'brokerageAccountAccessToken') {
+    if (
+      (type === 'brokerageAccountAccessToken' && !isTransferLink) ||
```
Comment (vitaliimovchan-front) [inline packages/front-b2b-link-rn/src/Front.tsx:60]: Web link does not ignore the `AccessTokenPayload` during TransferUX. I think both SDKs should be on the same page.

File: packages/front-b2b-link-rn/src/Front.tsx [hunk 2]
```diff
@@ -55,10 +56,12 @@ const FrontFinance = (props: {
...
+      type === 'transferFinished'
+    ) {
       setPayload(payload)
       props.onReceive?.(payload)
```
Comment (vitaliimovchan-front) [inline packages/front-b2b-link-rn/src/Front.tsx:64]: `onReceive` type for `transferFinished` should be `TransferFinishedPayload`

File: packages/front-b2b-link-rn/src/Front.tsx (hunk 1 above)
Comment (AlexZeml-Front) [reply packages/front-b2b-link-rn/src/Front.tsx]: fixed

File: packages/front-b2b-link-rn/src/Front.tsx (hunk 2 above)
Comment (AlexZeml-Front) [reply packages/front-b2b-link-rn/src/Front.tsx]: fixed

**Review by vitalii-movchan-mesh — APPROVED** [review]

**Pull request merged**